pkgs/by-name/ma/maxfetch/package.nix

```diff
@@ -0,0 +1,31 @@
+{ lib
+, stdenvNoCC
+, fetchFromGitHub
+}:
+
+stdenvNoCC.mkDerivation (finalAttrs: {
+  pname = "maxfetch";
+  version = "20230731";
+
+  src = fetchFromGitHub {
+    owner = "jobcmax";
+    repo = "maxfetch";
+    rev = "17baa4047073e20572403b70703c69696af6b68d";
+    hash = "sha256-LzOhrFFjGs9GIDjk1lUFKhlnzJuEUrKjBcv1eT3kaY8=";
+  };
+
+  installPhase = ''
+    runHook preInstall
+    install -Dm755 maxfetch $out/bin/maxfetch
```

Comment:
Why don't we wrap curl and all the other programs?

Reply:
I thought most of these programs would already be present on the end user's system, except curl maybe but that's only used in the update function which can't be used anyway.

I've wrapped gnused, ncurses (for tput) and procps (for ps). The only packages left out and curl, diffutils and gnupatch. These are only needed for updating.